src/core_modules/capture-core/components/CardList/CardList.component.js

```diff
@@ -35,67 +25,78 @@ const getStyles = (theme: Theme) => ({
     },
 });
 
-class CardList extends React.Component<Props> {
-    getDataElementsContainer: Function;
-    static defaultProps = {
-        itemTypeName: 'item',
-    }
-    constructor(props: Props) {
-        super(props);
-        this.getDataElementsContainer = makeDataElementsContainerSelector();
-    }
+const takeLastUpdatedEntry = statuses => statuses
+    .sort((a, b) => compareAsc(a.lastUpdated, b.lastUpdated))
```

Comment:
Yeah Ive tested it. I wil replace with moment after your suggestion tho

Reply:
UPDATE: this logic was there only because we wanted to give priority between canceled and completed actions. 
After the text suggestion you and markus made https://github.com/dhis2/capture-app/pull/875#discussion_r488609430 this logic is not serving any purpose anymore